examples/feeds/query_and_upload_feeds.py

```diff
@@ -0,0 +1,49 @@
+"""Functionality of uploading & querying feeds APIs."""
+
+from typing import Dict
+
+from nisystemlink.clients.core import ApiException, HttpConfiguration
+from nisystemlink.clients.feeds._feeds_client import FeedsClient
+from nisystemlink.clients.feeds.models import Platform
+
+
+# Constant
+FEED_NAME = "EXAMPLE FEED"
+FEED_DESCRIPTION = "EXAMPLE DESCRIPTION"
+PLATFORM = Platform.WINDOWS.value
+PACKAGE_NAME = ""
+PACKAGE_PATH = ""
+
+server_url = ""  # SystemLink API URL
+server_api_key = ""  # SystemLink API key
+workspace_id = ""  # Systemlink workspace id
+
+# Please provide the valid API key and API URL for client intialization.
+client = FeedsClient(HttpConfiguration(api_key=server_api_key, server_uri=server_url))
+
+# To upload a package to feed.
+try:
+    # To query available feeds.
+    query_feeds = client.query_feeds(
+        platform=PLATFORM,
+        workspace=workspace_id,
+    )
+    existing_feeds: Dict[str, str] = {}
+    feed_id = ""
+    for feed in query_feeds.feeds:
+        if feed.name == FEED_NAME and feed.id:
+            feed_id = feed.id
+            break
```

Comment:
It would be useful to create a utility to find a feed detail from list of feed details by feed name